lib/mpl_toolkits/axes_grid1/parasite_axes.py

```diff
@@ -93,7 +77,11 @@ def set_viewlim_mode(self, mode):
     def get_viewlim_mode(self):
         return self._viewlim_mode
 
+    @cbook.deprecated("3.4", alternative="apply_aspect")
     def update_viewlim(self):
+        return self._update_viewlim
```

Comment:
Shouldn't this be `self._update_viewlim()`?

Reply:
oops